### PR TITLE
Falco changes

### LINKED ISSUE #709: Small batch of falco adjustments
* make falling up air able to combo into shine on shield

* slight kb buff on late hit down air 

* alter laser to be more in line with melee

### PR DESCRIPTION
**Pull request by TeamPearson**
Resolves #709

### CODE REVIEW

File: fighters/falco/src/acmd/specials.rs [hunk 1]
```diff
@@ -131,7 +141,8 @@ unsafe fn falco_special_hi_game(fighter: &mut L2CAgentBase) {
         wait(lua_state, 1.0);
     }
     if is_excute(fighter) {
-        ATTACK(fighter, 0, 0, Hash40::new("hip"), 2.0, 75, 220, 0, 70, 6.0, 4.2, -3.1, -1.5, None, None, None, 1.5, 0.8, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_fire"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_KICK, *ATTACK_REGION_BODY);
+        println!("RUSH DEGREE: {}", WorkModule::get_float(fighter.boma(), *FIGHTER_FALCO_FIRE_STATUS_WORK_ID_FLOAT_TO_RUSH_DEGREE));
```
Comment (jobrien97) [inline fighters/falco/src/acmd/specials.rs]: Remove println

File: fighters/falco/src/acmd/specials.rs [hunk 2]
```diff
@@ -93,10 +97,16 @@ unsafe fn falco_special_air_s_end_game(fighter: &mut L2CAgentBase) {
  unsafe fn falco_special_hi_hold(fighter: &mut L2CAgentBase) {
         let lua_state = fighter.lua_state_agent;
         let boma = fighter.boma();
-    frame(lua_state, 20.0);
+        frame(lua_state, 1.0);
+        if is_excute(fighter) {
+            KineticModule::unable_energy_all(boma);
+        }
```
Comment (jobrien97) [inline fighters/falco/src/acmd/specials.rs]: This will make him unable to grab ledge during upB startup, when we implement it

**Review by jobrien97 — CHANGES_REQUESTED** [review]

File: fighters/falco/src/acmd/specials.rs (hunk 1 above)
Comment (techyCoder81) [reply fighters/falco/src/acmd/specials.rs]: this is still a draft PR lol

File: fighters/falco/src/acmd/specials.rs (hunk 2 above)
Comment (TeamPearson) [reply fighters/falco/src/acmd/specials.rs]: removed in favor of a param edit

**Comment by yoonikoo** [comment]
Changes are solid. Can gladly say that Falco is now more consistent.

**Comment by Riah321** [comment]
tired but it's good to go. ship it.

File: fighters/falco/src/acmd/specials.rs [hunk 3]
```diff
@@ -93,10 +97,15 @@ unsafe fn falco_special_air_s_end_game(fighter: &mut L2CAgentBase) {
  unsafe fn falco_special_hi_hold(fighter: &mut L2CAgentBase) {
         let lua_state = fighter.lua_state_agent;
         let boma = fighter.boma();
-    frame(lua_state, 20.0);
+        frame(lua_state, 1.0);
+        if is_excute(fighter) {
+        }
```
Comment (jobrien97) [inline fighters/falco/src/acmd/specials.rs]: doesn't need to be here

File: fighters/falco/src/acmd/specials.rs [hunk 4]
```diff
@@ -64,7 +64,7 @@ unsafe fn falco_special_air_n_loop_game(fighter: &mut L2CAgentBase) {
     if is_excute(fighter) {
         //DamageModule::add_damage(boma, 1.0, 0);
         WorkModule::on_flag(boma, *FIGHTER_FALCO_BLASTER_STATUS_WORK_ID_FLAG_LOOP_ACCEPT);
-        FT_MOTION_RATE(fighter, 5.0/(4.0-1.0));
+            FT_MOTION_RATE(fighter, 5.0/(4.0-1.0));
```
Comment (jobrien97) [inline fighters/falco/src/acmd/specials.rs]: extra indentation?

File: fighters/falco/src/acmd/specials.rs [hunk 5]
```diff
@@ -131,7 +140,8 @@ unsafe fn falco_special_hi_game(fighter: &mut L2CAgentBase) {
         wait(lua_state, 1.0);
     }
     if is_excute(fighter) {
-        ATTACK(fighter, 0, 0, Hash40::new("hip"), 2.0, 75, 220, 0, 70, 6.0, 4.2, -3.1, -1.5, None, None, None, 1.5, 0.8, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_fire"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_KICK, *ATTACK_REGION_BODY);
+       ("RUSH DEGREE: {}", WorkModule::get_float(fighter.boma(), *FIGHTER_FALCO_FIRE_STATUS_WORK_ID_FLOAT_TO_RUSH_DEGREE));
```
Comment (jobrien97) [inline fighters/falco/src/acmd/specials.rs]: remove println

**Review by jobrien97 — CHANGES_REQUESTED** [review]

File: fighters/falco/src/acmd/specials.rs (hunk 3 above)
Comment (TeamPearson) [reply fighters/falco/src/acmd/specials.rs]: removed

File: fighters/falco/src/acmd/specials.rs (hunk 4 above)
Comment (TeamPearson) [reply fighters/falco/src/acmd/specials.rs]: removed

File: fighters/falco/src/acmd/specials.rs (hunk 5 above)
Comment (TeamPearson) [reply fighters/falco/src/acmd/specials.rs]: removed

**Review by techyCoder81 — APPROVED** [review]

**Pull request merged**